juju/client/overrides.py

```diff
@@ -43,3 +48,35 @@ def __init__(self, deltas=None):
     @classmethod
     def from_json(cls, data):
         return cls(deltas=data)
+
+
+class ResourcesFacade(Type):
+    """Patch parts of ResourcesFacade to make it work.
+    """
+
+    @ReturnMapping(_client.AddPendingResourcesResult)
+    async def AddPendingResources(self, application_tag, charm_url, resources):
```

Comment:
I had discussed this with @mjs so there might be an issue already opened.  I'm following up to confirm or get one opened if not.